src/redux/sagas/extensions/extensionUpgrade.ts

```diff
@@ -4,15 +4,15 @@ import { ClientType, getExtensionHash } from '@colony/colony-js';
 
 import { ActionTypes } from '../../actionTypes';
 import { AllActions, Action } from '../../types/actions';
-import { putError, takeFrom, refreshExtension } from '../utils';
+import { putError, takeFrom, refreshExtensions } from '../utils';
```

Comment:
Super weird that github doesn't pick up the correct syntax highlight if you "rename"/"move" the file